R/process_extract_delayed_discharges.R

```diff
@@ -0,0 +1,87 @@
+#' Process the delayed discharges extract
+#'
+#' @description This will read and process the
+#' delayed discharges extract, it will return the final data
+#' but also write this out as a zsav and rds.
+#'
+#' @param year The year to process, in FY format.
+#' @param data The extract to process
+#' @param write_to_disk (optional) Should the data be written to disk default is
+#' `TRUE` i.e. write the data to disk.
+#'
+#' @return the final data as a [tibble][tibble::tibble-package].
+#' @export
+#' @family process extracts
+
+process_extract_delayed_discharges <- function(year, data, write_to_disk = TRUE) {
```

Comment:
```suggestion
process_extract_delayed_discharges <- function(data, year, write_to_disk = TRUE) {
```

I think it's just good practice to specify the data before the year